README.md

```diff
@@ -193,3 +193,61 @@ $ telnet 192.168.99.100 1234
 
 This `-p` argument links port 4321 on the container to port 1234 on the VM network adapter. It must appear after `docker run` and before `dmscid/plankton`. This allows us to connect to the container from outside of the VM, in this case using Telnet. The `192.168.99.100` IP is the IP of the VM on the bridge network between the host and the VM. VirtualBox will typically use this IP when available, but it may be different on your system.
 
+## Remote access to devices
+
+*Please note that this functionality should only be used on a trusted network.*
```

Comment:
No, I was more concerned with clients having direct access to the object on the server with no authentication. In fact this warning might apply to all our network functionality, but I hadn't thought about it so far, because the normal use case is that these things run in a container or local net anyway.